tests/rustdoc/issue-15318-2.rs

```diff
@@ -1,12 +0,0 @@
-// aux-build:issue-15318.rs
```

Comment:
Why delete this test?

Reply:
It expects `doc(primitive)` to work through an import.

Reply:
Shouldn't it be updated to ensure that it does not work instead?